docs/bits/keyword/letting_domain.md

```diff
@@ -13,7 +13,7 @@ Types of domains:
 - [boolean](https://github.com/conjure-cp/conjure/blob/main/docs/bits/type/L_bool.md)
 - [integer](https://github.com/conjure-cp/conjure/blob/main/docs/bits/type/L_int.md)
 - [enumerated](https://github.com/conjure-cp/conjure/blob/main/docs/bits/keyword/new_type_enum.md)
-- [unnamed](...)
+- [unnamed](https://github.com/conjure-cp/conjure/blob/main/docs/bits/type/L_unnamed.md)
```

Comment:
I think this shouldn't be L_unnamed. There is a lexeme called L_int, that's why I was suggesting renaming the other file.

Maybe just rename to unnamed?

Separately we should figure out why the vs-code extension doesn't show documentation for all lexemes, you might remember we noticed this the other day.

Reply:
I've  just renamed the file.